test/components/date-picker.spec.tsx

```diff
@@ -11,13 +13,20 @@ import {
     getNumOfPreviousDays
 } from '../../src/utils';
 
-const currentDate = 'CURRENT_DATE';
-const datePickerInputId = 'DATE_PICKER_INPUT';
-const datePickerDropdownId = 'DATE_PICKER_DROPDOWN';
+class DatePickerDemoDriver extends DriverBase {
+    public static ComponentClass = DatePickerDemo;
+
+    public get datePicker() {
+        return new DatePickerTestDriver(() => this.select('DATE_PICKER_DEMO', 'DATE_PICKER'));
```

Comment:
every time this getter will be called it will create a new `DatePickerTestDriver`. I think you can just make it into a public variable.

Reply:
also: return type